### PR TITLE
Overall and Ethics Learning Objectives 

### PR DESCRIPTION
**Pull request by carriewright11**
adding new objectives to ethics chapter and updating overall objectives to avoid the word "understand" etc.

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -43,12 +43,12 @@ This course is available in multiple formats which allows you to take it in the
...
-- Learn how to write and debug code using AI-based tools.
-- Explore the benefits of refactoring code using AI-powered techniques, such as making code readable, keeping it brief, and optimizing code.
-- Understand the importance of annotating code, and learn best practices for doing so using AI.
-- Learn how to use AI-based tools for understanding and analyzing code, such as code comprehension of unfamiliar languages or functions.
-- Explore the challenges and ethical implications of using AI for different aspects of software development.
+- Explain the basics of AI and its potential for improving software development workflows.
+- Implement strategies to write and debug code using AI-based tools.
+- Describe the benefits of refactoring code using AI-powered techniques, such as making code readable, keeping it brief, and optimizing code.
+- Apply best practices for annotating code using AI.
+- Recognize strategies for how to use AI-based tools for understanding and analyzing code, such as code comprehension of unfamiliar languages or functions.
```
Comment (avahoffman) [inline index.Rmd:50]:
```suggestion
- Recognize strategies for using AI-based tools to understand and analyze code, such as code comprehension of unfamiliar languages or functions.
```

File: quizzes/quiz_ch2.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (avahoffman) [inline quizzes/quiz_ch2.md:1]: Doesn't this need to be in a separate repo?

**Review by avahoffman — APPROVED** [review]

**Review by ehumph — APPROVED** [review]
LGTM

File: quizzes/quiz_ch2.md (hunk 2 above)
Comment (carriewright11) [reply quizzes/quiz_ch2.md:1]: @cansavvy  knows more about that if you want to talk more about this

**Comment by carriewright11** [comment]
I'll wait to merge this till I hear about the plans for the location of quizzes

File: quizzes/quiz_ch2.md (hunk 2 above)
Comment (cansavvy) [reply quizzes/quiz_ch2.md:1]: You can have quizzes in the same repo if you don't care that the quiz answers will be public.

Comment (carriewright11) [reply quizzes/quiz_ch2.md:1]: I believe the thought was that if someone tried hard enough to track down the repo etc. then they were trying pretty hard and that would be rare - I wonder if we could obscure them a tad though... like name the dir something less obvious to learners but still reasonably findable to developers?

Comment (cansavvy) [reply quizzes/quiz_ch2.md:1]: Yeah, this question generally depends on who is taking this course and why? Will it be something they will be required to take for certification or will they just be taking for funsies?

Comment (carriewright11) [reply quizzes/quiz_ch2.md:1]: it would be nice to make that more flexible for OTTR though - I will add this to an issue

Comment (carriewright11) [reply quizzes/quiz_ch2.md:1]: it's hard to answer that question really because people use coursera for lots of reasons

Comment (carriewright11) [reply quizzes/quiz_ch2.md:1]: what do you think @avahoffman  and @ehumph ?

**Comment by carriewright11** [comment]
Maybe we can merge this and then move the quizzes later if we decide to do that.

**Pull request merged**